src/core_plugins/elasticsearch/lib/migrate_config.js

```diff
@@ -1,24 +1,18 @@
-import { get } from 'lodash';
 import upgrade from './upgrade_config';
+import { SavedObjectsClient } from '../../../server/saved_objects';
 
-export default function (server, { mappings }) {
+export default async function (server, { mappings }) {
```

Comment:
`upgrade` is using the server for `server.log` as well as `server.config()`, so I can't remove passing the `server` to these functions; however, I am now passing the `savedObjectsClient` to the `upgrade` function so it's only being constructed once.